packages/base-docs/src/doc-plugin.ts

```diff
@@ -106,7 +106,7 @@ export class DocPlugin extends Plugin {
         // this._markDocAsFocused();
     }
 
-    initializeCommands(): void {
+    private initializeCommands(): void {
```

Comment:
Please add a prefix underbar to private members of a class.

Reply:
I've talked to you before about whether it's necessary to add an underline prefix to a private method, so let's add a prefix uniformly